components/tree/Tree.razor.cs

```diff
@@ -687,26 +687,26 @@ public TreeNode<TItem> FindFirstOrDefaultNode(Func<TreeNode<TItem>, bool> predic
         /// </summary>
         public void ExpandAll()
         {
-            this.ChildNodes.ForEach(node => Switch(node, true));
+            ChildNodes.ForEach(node => _ = Switch(node, true));
         }
 
         /// <summary>
         /// Collapse all nodes
         /// </summary>
         public void CollapseAll()
         {
-            this.ChildNodes.ForEach(node => Switch(node, false));
+            ChildNodes.ForEach(node => _ = Switch(node, false));
         }
 
         /// <summary>
         /// 节点展开关闭
         /// </summary>
         /// <param name="node"></param>
         /// <param name="expanded"></param>
-        private void Switch(TreeNode<TItem> node, bool expanded)
+        private async Task Switch(TreeNode<TItem> node, bool expanded)
```

Comment:
Are the `Switch` and `SwitchAllNodes` duplicated?

Reply:
Yes, that's fixed.